metashare/api/jobs.py

```diff
@@ -338,3 +338,27 @@ def refresh_commits(*, repository, branch_name):
 
 
 refresh_commits_job = job(refresh_commits)
+
+
+def populate_github_users(repository):
+    user = repository.get_a_matching_user()
+    if user is None:
+        logger.warning(f"No matching user for repository {repository.pk}")
+        return
+    repo = get_repo_info(user, repository.repo_id)
+    repository.github_users = [
+        {
+            "id": collaborator.id,
+            "login": collaborator.login,
+            "avatar_url": collaborator.avatar_url,
+        }
+        for collaborator in repo.collaborators()
+    ]
+    # We don't use the finalize_* pattern here because the frontend
+    # doesn't subscribe to Repositories. Maybe we should re-evaluate
+    # this, but for now, we assume that Repositories don't refresh
+    # GitHub users frequently enough to merit a websocket push.
```

Comment:
I'm okay with that -- maybe we add a subtle "refresh users" button to the UX? That would seem appropriate, if you've e.g. just messed with repo permissions.

Reply:
Created a new story for this: https://trello.com/c/5p5Scm9H